.rubocop.yml

```diff
@@ -0,0 +1,30 @@
+AllCops:
+  TargetRubyVersion: 2.5
+  NewCops: disable
+  Exclude:
+    # exclude everything except lib dir
+    - !ruby/regexp /^(?!lib\/)/
```

Comment:
Creative. So what is this value documented to be exactly?

Reply:
Ah sadly that Regexp didn't work. So I got more creative 😂 

Check the latest, I'm 95% confident this will work on Evergreen as well.